seed/views/00301_team_case_overview.js

```diff
@@ -0,0 +1,26 @@
+exports.seed = function (knex, promise) {
+  var sql = `CREATE VIEW app.team_case_overview
+  WITH SCHEMABINDING
+  AS
+  SELECT
+      MAX(CONCAT(om.forename, ' ', om.surname)) AS name
+    , MAX(iv.grade_code) AS grade_code
+    , MAX(iv.total_cases) AS total_cases
+    , MAX(iv.available_points) AS available_points
+    , MAX(iv.total_points) AS total_points
+    , MAX(iv.contracted_hours) AS contracted_hours
+    , MAX(iv.default_contracted_hours_po) AS default_contracted_hours_po
+    , MAX(iv.default_contracted_hours_pso) AS default_contracted_hours_pso
+    , MAX(iv.reduction_hours) AS reduction_hours
+    , MAX(iv.team_id) AS id
+    , MAX(iv.workload_owner_id) AS link_id
```

Comment:
Just a comment: Not thrilled about all these MAX.  Could be better as a max on the workload_owner_id in a subquery which is then joined back to the rest of the stuff: eg Select id, stuff from places Join (select max(id) from there) on id=id.  

Would that work? Would it be easier to understand? (only asking, don't change)

Reply:
Agree, it's not particularly nice. Discussed in person, going to leave for now